libecs/source/Core/Scene.cxx

```diff
@@ -23,50 +20,35 @@ std::string Scene::getName() { return name; }
 
 Entity Scene::CreateEntity()
 {
-    Entity newEntity = mEntityManager->CreateEntity();
-    mComponentManager->AddComponent(newEntity, "Entity " + std::to_string(newEntity), mTagId);
+    Entity newEntity = mEntityManager.CreateEntity();
+    mComponentManager.AddComponent(newEntity, "Entity " + std::to_string(newEntity), mTagId);
     return newEntity;
 }
 
 Entity Scene::CreateEntity(std::string newName)
 {
-    Entity newEntity = mEntityManager->CreateEntity();
-    mComponentManager->AddComponent(newEntity, data::Value{data::Record{{"name", newName}}}, mTagId);
+    Entity newEntity = mEntityManager.CreateEntity();
+    mComponentManager.AddComponent(newEntity, data::Value{data::Record{{"name", newName}}}, mTagId);
     return newEntity;
 }
 
-std::unordered_map<Entity, Signature> Scene::getEntities() { return mEntityManager->getEntities(); }
+std::unordered_map<Entity, Signature> Scene::getEntities() { return mEntityManager.getEntities(); }
 
 void Scene::DestroyEntity(Entity entity)
 {
-    mEntityManager->DestroyEntity(entity);
-    mComponentManager->EntityDestroyed(entity);
-    mSystemManager->EntityDestroyed(entity);
+    mEntityManager.DestroyEntity(entity);
+    mComponentManager.EntityDestroyed(entity);
 }
 
-Signature Scene::getSignature(Entity entity) { return mEntityManager->GetSignature(entity); }
+Signature Scene::getSignature(Entity entity) { return mEntityManager.GetSignature(entity); }
```

Comment:
Prefer returning refs
```suggestion
const Signature &Scene::getSignature(Entity entity)const { return mEntityManager.GetSignature(entity); }
```

Reply:
This will be removed in the next cleanup.